suites/pacific/rados/tier-2_rados_test-pg-split-merge.yaml

```diff
@@ -0,0 +1,105 @@
+# Suite contains tests related to verify slow op requests
```

Comment:
Please update the intent of the suite file.

Reply:
done